packages/core/src/_common/storage.js

```diff
@@ -140,14 +140,15 @@ const CookieStorage = function(cookie_name, cookie_domain) {
 
     this.initialized = false;
     this.cookie_name = cookie_name;
+    // 'app.test.domain' and 'test.domain' are used during dev as hostnames, for deriv-app and deriv-com, respectively
+    // To do so simply change host to 'app.test.domain' in webpack settings in deriv-app
+    // and run 'gatsby develop -H 0.0.0.0' in deriv-com
+    // Note. You need to point both sample domains to localhost (127.0.0.1)
```

Comment:
This is referring to Gatsby where it is not used in this repo. Please comment in the appropriate repository.